lib/matplotlib/backend_managers.py

```diff
@@ -234,7 +234,7 @@ def remove_tool(self, name):
 
         del self._tools[name]
 
-    def add_tool(self, name, tool, *args, **kwargs):
+    def add_tool(self, tool_name):
```

Comment:
don't remove the *args, and **kwargs, those are really useful when adding custom tools. 
Sometimes you need to pass initialization arguments when manually adding tools

Reply:
yup, restored it